app/views/links/find.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: "links", object: @link, as: 'link' %>
```

Comment:
Wait ... does `render` take an `as: ` option? I don't think it does which means this code is a little buggy.

I think what you probably want is to use locals which is a little easier to understand than the `object: @foo` nonsense. You could use locals like so: `render partial: "links", locals: { link: @link }`.